### ISSUE TITLE
Clean command

### ISSUE THREAD

**Issue by kevinresol** [lix-pm/lix.client#14]
I use lix to deploy to heroku but heroku only provides mechanism to ignore (remove) files _at the very beginning of a build_. As a result, my deployed container image includes all the downloaded haxe libraries. It would be nice if lix has a command to delete everything it that it has downloaded, which can be called after haxe compiles.

**Comment by back2dos** [comment]
I assume you'll want to delete the downloaded haxe as well then?

**Comment by kevinresol** [comment]
Yeah, but I don't mind having some switches to distinguish libs and the haxe binaries

**Comment by back2dos** [comment]
Well, I mean `rm -rf ~/haxe` will basically do the trick if the granularity of a Tsar Bomba will do for you ;)

**Comment by kevinresol** [comment]
Sure, if then maybe lix can provide some commands that returns the path to haxe or the downloaded libraries. That would ease the pain of figuring out the path by digging through docs and source codes :smile:
